package/wpe/wpeframework/wpeframework.mk

```diff
@@ -4,8 +4,8 @@
 #
 ################################################################################
 
-WPEFRAMEWORK_VERSION = aed13867c93fe76dc29d32abe4237559cc19b62c
-WPEFRAMEWORK_SITE = $(call github,WebPlatformForEmbedded,WPEFramework,$(WPEFRAMEWORK_VERSION))
+WPEFRAMEWORK_VERSION = 2c1a62d177473c67c1abbd41147a051edb436433
+WPEFRAMEWORK_SITE = $(call github,s-rapolu,WPEFramework,$(WPEFRAMEWORK_VERSION))
```

Comment:
Same here, cant point the entire buildroot to your fork. Needs to point to master with your changes merged in on the main repo.

Reply:
Sorry, corrected the mistake and now pointing it to master.